Xwt.XamMac/Xwt.Mac/TableViewBackend.cs

```diff
@@ -446,6 +446,19 @@ class TableRowView : NSTableRowView
 				// may be drawn already and it will not be redrawn even
 				// if Selection has been changed.
 				NeedsDisplay = true;
+				// Since Big Sur NSTableView doesn't refresh its subviews on selection change
+				if (MacSystemInformation.OsVersion >= MacSystemInformation.BigSur)
```

Comment:
This is very very weird, expensive and horrible.

Reply:
Isn't it because of the Mojave change with layer backed views?
https://developer.apple.com/documentation/macos-release-notes/appkit-release-notes-for-macos-10_14

Reply:
> Views that implicitly depend on being redrawn when an ancestor, descendant, or intersecting sibling is redrawn may not be redrawn. As before, if a view needs to be redrawn, set its needsDisplay property to true.

Specifically.

Reply:
No, this can be only observed on BigSur, also no layers involved here. I have no idea how to make this in a better way. Ideas?

Reply:
Only thing I can find: https://developer.apple.com/forums/thread/659488

Reply:
Filed an internal bug for bad code. Per discussion on Teams let's merge this.